apps/web-app/components/convos/convoMessageItem.vue

```diff
@@ -57,14 +57,11 @@
   });
 
   const typeClasses = computed(() => {
-    if (userIsAuthor.value) {
-      return 'bg-blue-10 text-white';
-    }
```

Comment:
we need to keep this to mark the user's own messages in blue
i would suggest copying the below `switch / case` block here to only change the styling if the current user is the author of the message